2020-02-Spring/1_strings/README.md

```diff
@@ -141,21 +157,35 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s3"/>
 
-### 3. SOLUTION 3 TODO :bug:
+### 3. SOLUTION Add String
 
-Source: TODO :bug:
-
-#### Naive/Simple Solution 
-
-TODO :bug:
 
 #### Optimal Solution
 
-TODO :bug:
-
-#### Testing The Solutions OR Driver For Solution
+C++ solution
+
```

Comment:
If you found the written explanations of solution strategies at all helpful as an attendee, I'd recommend doing that here. it also greatly helps your comprehension of the problem and will help you explain solution strategies in general. (This is important for leading discussions about these problems, as well as interviews in real life.)